lib/cucumber/rake/task.rb

```diff
@@ -113,7 +113,15 @@ def run
       attr_reader :cucumber_opts
 
       def cucumber_opts=(opts) # :nodoc:
-        @cucumber_opts = String == opts.class ? opts.split(' ') : opts
+        unless String == opts.class
```

Comment:
This should be opts.class == String (In rubocop it's referred to as yodacondition)

Reply:
Will change it
I actually copy/paste that from somewhere else in the code. Will take a look if there are a lot like that or not, maybe change all at once

Reply:
Done